website/docs/cluster-management/managing-clusters-without-capi.mdx

```diff
@@ -154,7 +153,13 @@ Here's how to create a kubeconfig secret.
 
 	</details>
 
-5. Obtain the cluster certificate (CA). How you do this depends on your cluster. If you're using GKE, for example, you can view the CA on the GCP Console: Cluster->Details->Endpoint->”Show cluster certificate”. You'll need to copy the contents of the certificate into the `ca.crt` file used below.
+5. Obtain the cluster certificate (CA). How you do this depends on your cluster.
```

Comment:
should these changes go in `/version-0.30.0` as well?

Reply:
Done